fusionauth-site.iml

```diff
@@ -18,7 +18,7 @@
     <orderEntry type="library" scope="PROVIDED" name="algolia_html_extractor (v2.6.4, rbenv: 2.7.0) [gem]" level="application" />
     <orderEntry type="library" scope="PROVIDED" name="algoliasearch (v1.27.5, rbenv: 2.7.0) [gem]" level="application" />
     <orderEntry type="library" scope="PROVIDED" name="asciidoctor (v2.0.10, rbenv: 2.7.0) [gem]" level="application" />
-    <orderEntry type="library" scope="PROVIDED" name="bundler (v2.2.5, rbenv: 2.7.0) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="bundler (v2.2.20, rbenv: 2.7.0) [gem]" level="application" />
```

Comment:
I don't know about changing this file. It's not something I normally do.

Reply:
Would suggest rolling it back unless there's some reason to change it.

Reply:
This is a file we fight with a lot, I think it has to do with the version of ruby you have installed. In theory the `setup.sh` should set up the correct version of ruby and bundler, etc. 

Perhaps we still have an issue here. @voidmain is the expert here, I just pretend to follow along.

Reply:
I've tried to fix the bundler version in `setup.sh`, but I might have not got it perfect yet. @matt1hathcock - you should run that again and see if it fixes your bundler version.

Reply:
@voidmain @robotdan nothing changed when I ran `/.setup.sh` again. I did this a while back and forgot to reply. The fusionauth-site.iml seems to have changes any time I run `sb serve`